federation/apis/federation/doc.go

```diff
@@ -1,5 +1,3 @@
-// +build !ignore_autogenerated
```

Comment:
@thockin - why we don't want to have DeepCopy to be generated fo pkg/util/intstr? It seems useful.